packages/code-du-travail-ui/src/ToggleButton/index.js

```diff
@@ -41,133 +22,69 @@ ToggleButton.defaultProps = {
 export default ToggleButton;
 
 const Button = styled.button`
-  user-select: none;
-  appearance: none;
-  cursor: pointer;
-  display: inline-block;
-  margin: 0;
+  padding: ${spacing.small} ${spacing.base};
   text-align: center;
-  white-space: nowrap;
   line-height: inherit;
   font-size: inherit;
   font-weight: 600;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-rendering: optimizeLegibility;
-
-  text-decoration: none;
-
-  padding: ${spacing.small} ${spacing.base};
-  border-radius: ${box.borderRadius};
   border: 1px solid transparent;
   border-bottom-width: 2px;
   border-top-width: 1px;
+  border-radius: ${box.borderRadius};
+  cursor: pointer;
+  transition: all 250ms ease;
```

Comment:
y'a une logique derriere la ré-orientation des rêgles ?
Ca rajoute beaucoup de bruit pour la relecture :(

Reply:
Alors il y a une raison mais c'est relativement personnel. J'ai une sorte de "modèle" dans ma tête pour les règles: 
1- Tout ce qui est display et positionnement (display: flex, z-index, position absolute etc.)
2 - Marges / padding
3 - Tout ce qui est typo
4 - Tout ce qui est visuel (background first, then borders, then box shadows etc)
5 - d'éventuels trucs restants

Cet article est pas loin de mon approche https://webdesign.tutsplus.com/articles/outside-in-ordering-css-properties-by-importance--cms-21685 (on n'est pas d'accord sur le z-index apparement et lui met typo pratiquement en dernier)

Ils y en a qui préfèrent pas ordre alphabétique mais j'ai vraiment pas l'habitude de ça

Reply:
L'avantage de ça c'est qu'une fois que tu prends l'habitude du "scan" très rapidement les css et tu visualises facilement ce à quoi doit ressembler l'element.

Reply:
pas de problème, c'est plutôt une bonne idée de faire ça, 
il faut qu'on document ça quelquepart.

Reply:
stylelint ? ca existe pour les styled-components ?

Reply:
J'ouvre une issue pour ça. Je sais qu'on en avait sur un ancien projet mais pas sur que ce soit adaptable aux styled-components